primer/brand
# Updates to form examples, inline link and form control components

## Summary

Updates to form recipes, form control and inline link components.

🔗 [See form example changes in Storybook](https://primer-883531b99b-26139705.drafts.github.io/storybook/?path=/story/recipes-feature-previews-level-1--level-one-side-by-side)
🔗 [See form control changes in Storybook](https://primer-883531b99b-26139705.drafts.github.io/storybook/?path=/story/components-forms-formcontrol--text-input-playground)
🔗 [See inline link changes in Storybook](https://primer-883531b99b-26139705.drafts.github.io/storybook/?path=/story/components-inlinelink--size-inheritence)

## List of notable changes:



- **updated** InlineLink underline to now work across multiple lines.
- **updated** InlineLink dark mode colors to match recent changes to Link
- **updated** FormControl labels from 14px to 16px to match labels in Figma
- **updated** Side by side form example recipes to match latest designs

## What should reviewers focus on?

- Review the side by side story example, and toggle between `default` and `extended` form type controls to see different versions.
- Check inline link color values are correct and appear correctly across multiple lines

## Steps to test:



🔗 [See form example changes in Storybook](https://primer-883531b99b-26139705.drafts.github.io/storybook/?path=/story/recipes-feature-previews-level-1--level-one-side-by-side)
🔗 [See form control changes in Storybook](https://primer-883531b99b-26139705.drafts.github.io/storybook/?path=/story/components-forms-formcontrol--text-input-playground)
🔗 [See inline link changes in Storybook](https://primer-883531b99b-26139705.drafts.github.io/storybook/?path=/story/components-inlinelink--size-inheritence)

## Supporting resources (related issues, external links, etc):

- [Reference designs in Figma](https://www.figma.com/file/HIEQHWa8Cuw4AZfFisALJV/Contentful-Templates?type=design&node-id=1262-3386&mode=design&t=oDodcfwcQEwU6fJk-4)


## Contributor checklist:


## Reviewer checklist:


## Screenshots:



Link colors in dark mode:

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">


![Screenshot 2023-09-25 at 13 10 39](https://github.com/primer/brand/assets/13340707/5463e0c3-2c49-4d35-844f-58007cd9948c)

 </td>
<td valign="top">

![Screenshot 2023-09-25 at 13 11 15](https://github.com/primer/brand/assets/13340707/2ccb81d8-c01e-40d2-a800-036349d983ba)


</td>
</tr>
</table>


Link underline across multiple lines:

<table>
<tr>
<th> Before</th> <th> After </th>
</tr>
<tr>
<td valign="top">

![Screenshot 2023-09-26 at 11 44 39](https://github.com/primer/brand/assets/13340707/9b768a5a-766a-44be-b453-1c42e5d5298d)



 </td>
<td valign="top">

![Screenshot 2023-09-26 at 11 44 53](https://github.com/primer/brand/assets/13340707/47392cfd-aa5d-46d5-b631-a3a34c39bc2d)


</td>
</tr>
</table>

New form examples:


https://github.com/primer/brand/assets/13340707/8ad4e1fb-2dde-4b86-9c2c-4895647088a0
